diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 62747b5714ed02fee4c9df7c55b28d95f3ac2a46..2b0f6ca51e39ac1cca91037aed056cf6e5016b63 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -134,6 +134,7 @@ static_library("renderer") {
   public_deps = [ "//components/contextual_search:buildflags" ]
 
   deps = [
+    "//brave/renderer",
     "//chrome:resources",
     "//chrome:strings",
     "//chrome/common",
