diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index e6496ae6ef9f22325338c91309444e0bd553e9d8..4478a34a9b50aa3a21664066e8a2c0e895850899 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("channel_info") {
   }
 
   deps = [ "//build:branding_buildflags" ]
+  if (brave_chromium_build) { deps += [ "//brave/common:channel_info" ] }
 
   public_deps = [
     "//base",
@@ -168,6 +169,7 @@ static_library("common") {
   ]
 
   public_deps = [
+    "//brave/common",
     ":available_offline_content_mojom",
     ":buildflags",
     ":channel_info",
@@ -270,7 +272,7 @@ static_library("common") {
   if (enable_plugins) {
     public_deps += [ "//ppapi/shared_impl" ]
   }
-
+  sources += brave_chrome_common_sources public_deps += brave_chrome_common_public_deps
   if (enable_extensions) {
     sources += [
       "cast_messages.cc",
@@ -542,6 +544,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
