diff --git a/components/translate/core/browser/BUILD.gn b/components/translate/core/browser/BUILD.gn
index 0c629705b6dd370611bbc30db1d78e70e1a51d3e..46df38ca9402da64a497c52ba21dd22644bdacb3 100644
--- a/components/translate/core/browser/BUILD.gn
+++ b/components/translate/core/browser/BUILD.gn
@@ -82,6 +82,7 @@ static_library("browser") {
     ]
     deps += [ "//components/infobars/core" ]
   }
+  deps += ["//brave/components/translate/core/browser"]
 }
 
 source_set("translate_pref_names") {
@@ -104,6 +105,7 @@ source_set("unit_tests") {
     "translate_script_unittest.cc",
     "translate_ui_delegate_unittest.cc",
     "translate_url_util_unittest.cc",
+    "../../../brave/chromium_src/components/translate/core/browser/brave_translate_manager_unittest.cc",
   ]
   deps = [
     ":browser",
